src/nile/base_project/.env

```diff
@@ -0,0 +1 @@
+# ALIAS = PRIVATE_KEY
```

Comment:
let's add an actual private key, and a working file

Reply:
Should we keep it commented like: `# PRIVATE_KEY = 0xe3e70682c2094cac629f6fbed82c07cd`?

And what do you mean by a working file?

Reply:
i mean that it's ready to be used. why commenting the key?

Reply:
Ohhhh my mistake, I understand. So let's not comment the key :)

Reply:
are `0x` keys currently supported?

Reply:
`0x` keys are indeed supported